kpi/tests/test_mock_data_exports.py

```diff
@@ -454,6 +454,16 @@ def test_xls_export_filter_fields(self):
         ]
         self.run_xls_export_test(expected_lines, export_options)
 
+    def test_xls_export_filter_fields_without_index(self):
+        export_options = {'fields': ["start", "end", "Do_you_descend_from_unicellular_organism"]}
+        expected_lines = [
+            [ "start", "end", "Do you descend from an ancestral unicellular organism?" ],
+            [ "2017-10-23T05:40:39.000-04:00", "2017-10-23T05:41:13.000-04:00", "No" ],
+            [ "2017-10-23T05:41:14.000-04:00", "2017-10-23T05:41:32.000-04:00", "No" ],
+            [ "2017-10-23T05:41:32.000-04:00", "2017-10-23T05:42:05.000-04:00", "Yes" ],
```

Comment:
Not a blocker at all, Just a notice that we could remove the leading and trailing spaces in each list ( and use single quotes ;-)). (Several other places in this file too)